CHANGELOG.md

```diff
@@ -2,6 +2,106 @@
 
 ## Unreleased
 
+This release is a notable release for metrics:
+
+* The base2 exponential histogram aggregation has been marked as stable. To use, configure
+  your `MetricExporter` with a default aggregation
+  of `Aggregation.base2ExponentialBucketHistogram()` for histogram instruments. If using OTLP
+  exporter with autoconfigure,
+  set `OTEL_EXPORTER_OTLP_METRICS_DEFAULT_HISTOGRAM_AGGREGATION=BASE2_EXPONENTIAL_BUCKET_HISTOGRAM`.
+  If using configuring OTLP programatically,
+  use `Otlp*MetricExporterBuilder#setDefaultAggregationSelector(DefaultAggregationSelector)`.
+* The metrics SDK undergone significant internal refactoring which results in reduced complexity and
+  significantly reduced memory allocation during metric collection / export.
+
+### API
+
+#### Logs
+
+* WARNING: Split out Event API from Log API. Event API is now published in `opentelemetry-api-logs`.
+  API / SDK usage has also changed - see PR for details.
+  ([#5049](https://github.com/open-telemetry/opentelemetry-java/pull/5049))
+
+### SDK
+
+* Add shutdown / close to OpenTelemetrySdk
```

Comment:
Minor nit: do we want to use code-quotes for all class/methods names? Some of them are formatted, some of them note
```suggestion
* Add shutdown / close to `OpenTelemetrySdk`
```

Reply:
Good point. Added formatting to all the places it was missing.